lib/chef/knife/bootstrap/windows-chef-client-msi.erb

```diff
@@ -191,9 +191,9 @@ echo Writing validation key...
 echo Validation key written.
 @echo on
 
-<% if @config[:encrypted_data_bag_secret] -%>
+<% if @config[:secret] -%>
 > <%= bootstrap_directory %>\encrypted_data_bag_secret (
- <%= encrypted_data_bag_secret %>
```

Comment:
@adamedx  We have taken care of above in this PR Please review : https://github.com/ClogenyTechnologies/knife-windows/commit/b57512bca48015b6d92a4dc320056737d36ce9df